### PR TITLE
Keep Workplaces Active

### PR DESCRIPTION
**Pull request by chadtomkiss-madetech**
**Work done**

- Added ability for Admins to email workplaces who have been inactive for 6/12/18/24 months

#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/features/search/emails/emails.component.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              })),
+            );
+          }),
+        )
+        .subscribe(({ latestCampaign, inactiveWorkplaces }) => {
+          this.history.unshift(latestCampaign);
+
+          this.alertService.addAlert({
+            type: 'success',
+            message: `${this.decimalPipe.transform(latestCampaign.emails)} emails sent successfully.`,
```
Comment (chadtomkiss-madetech) [inline src/app/features/search/emails/emails.component.ts:70]: I wonder if this should say "scheduled to be sent", as Send in Blue only sends 50 emails per hour

File: server/routes/admin/email-campaigns/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const express = require('express');
+const router = express.Router();
+
+router.use('/inactive-workplaces', require('./inactive-workplaces'));
+
+module.exports = router;
```
[the remark this answers is not in the document]
Comment (chadtomkiss-madetech) [reply server/routes/admin/email-campaigns/index.js:6]: @aaron-russell Hmm.. I think the `require()` inline communicates it's a nested route, as most vars are functions in the same file that handles the endpoint.

We do the same for bulk upload routes (but that was me, so maybe that needs changing too 🙈 )

File: server/routes/admin/email-campaigns/inactive-workplaces/report.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    workplaceId: item.nmdsId,
+    lastUpdated: item.lastUpdated,
+    emailTemplate: item.emailTemplate.name,
+    dataOwner: item.dataOwner,
+    nameOfUser: item.user.name,
+    userEmail: item.user.email,
+  });
+};
+
+const generateReport = async (_, res) => {
```
[the remark this answers is not in the document]
Comment (chadtomkiss-madetech) [reply server/routes/admin/email-campaigns/inactive-workplaces/report.js:19]: It communicates that `req` isn't used anywhere in the function.

Not sure if that's ok in JS - it's a convention used in Python/Go

File: server/services/email-campaigns/inactive-workplaces/findInactiveWorkplaces.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+const transformInactiveWorkplaces = (inactiveWorkplace) => {
+  const emailTemplate = nextEmail.getTemplate(inactiveWorkplace);
+
+  return {
+    id: inactiveWorkplace.EstablishmentID,
+    name: inactiveWorkplace.NameValue,
+    nmdsId: inactiveWorkplace.NmdsID,
+    lastUpdated: inactiveWorkplace.LastUpdated,
+    emailTemplate: emailTemplate,
```
Comment (aaron-russell) [inline server/services/email-campaigns/inactive-workplaces/findInactiveWorkplaces.js:15]: You can just use it once in ES6+ 
```suggestion
    emailTemplate,
```

File: server/routes/admin/email-campaigns/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (aaron-russell) [reply server/routes/admin/email-campaigns/index.js:6]: Yeah, but in this PR, you also break this convention

**Review by aaron-russell — APPROVED** [review]

**Pull request merged**